doc/spec/spec.rst

```diff
@@ -8,8 +8,8 @@ This is the spec.
     api
     estimation
     inference
-    benchmark
```

Comment:
There's another build break because now the benchmark.rst file isn't referenced from anywhere else; please delete the .rst file to correspond to this change.

Reply:
Done